src/js/utils/HubClient.js

```diff
@@ -8,7 +8,8 @@ var assetStore = require('./asset-store');
 var connectionCache = require('./connection-cache');
 var NodeListGeneration = require('./scene-graph/node-list-generation');
 var GraphTypes = require('../constants/graph-constants').GraphTypes;
-var CategoryConfig = require('../utils/scene-graph/category-config-generator')
+var CategoryConfig = require('../utils/scene-graph/category-config-generator');
+var GraphThemeGenerator = require('../utils/scene-graph/graphTheme-generator');
```

Comment:
A tiny point but the new file name would probably suit graph-theme-generator ? just following largely precedent (yes we do have a few exceptions but they are largely old files)